Bottom: 6effe94308edae9f441d27bf85ed74ebdd86b085
Top:    08b28bf8267575e63cb08f4ef4a55acf86d42e54
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:57:50 +0000

polygon1.c: Remove now unused memory allocation macro MemGet


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 7b3e396..ecbdddd 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -105,10 +105,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 //#define DEBUG_INTERSECT
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
